dashboard/app/views/layouts/_unsupported_browser.html.haml

```diff
@@ -18,15 +19,18 @@
     if (isUnsupportedBrowser()) {
       textDivId = '#unsupported-browser';
     } else if (typeof appOptions !== 'undefined') {
-      if (isSmallDevice()) {
-        if (appOptions.app == 'applab') {
+      if (isMobileDevice()) {
+        if (appOptions.app === 'applab') {
           textDivId = '#applab-unsupported-tablet';
-        } else if (appOptions.app == 'gamelab') {
+        } else if (appOptions.app === 'gamelab') {
           textDivId = '#gamelab-unsupported-tablet';
         }
         dismissable = true;
-      } else if (isIE11() && appOptions.app == 'weblab') {
-        textDivId = '#weblab-unsupported-browser'
+      } else if (isIE11() && appOptions.app === 'weblab') {
+        textDivId = '#weblab-unsupported-browser';
+        dismissable = true;
+      } else if (!isStorageAvailable('localStorage') && appOptions.app === 'weblab') {
```

Comment:
minor comment, but since the `isStorageAvailable()` call actually writes to local storage, I'd flip the order and check `appOptions.app === 'weblab'` first so you don't need to do the `isStorageAvailable()` call for any other apps.